src/atoms/Layout/index.js

```diff
@@ -81,7 +82,8 @@ function processChildren(props) {
       largeCols: largeCols[lgIdx],
       xLargeCols: xLargeCols[xlgIdx],
       xxLargeCols: xxLargeCols[xxlgIdx],
-      bottomSpacing
+      bottomSpacing,
+      fullwidth: fullWidthAll,
```

Comment:
@patrickkim bhaha - - you had to go camelCase on the fullWidth. If you're going to do this, just change the other one so it's consistent. 

[fullwidth](https://en.wiktionary.org/wiki/fullwidth) is a word, though, and I think it looks much cleaner. Camel case is super ugly and harder to read.